service/resource/cloudformationv2/adapter/adapter.go

```diff
@@ -26,11 +26,11 @@
 package adapter
```

Comment:
For my interest, what is the idea behind the adapter package here? What problem does it solve?